llvm/lib/Target/RISCV/MCTargetDesc/RISCVBaseInfo.h

```diff
@@ -326,6 +326,7 @@ enum OperandType : unsigned {
   OPERAND_SIMM12,
   OPERAND_SIMM12_LSB00000,
   OPERAND_SIMM26,
+  OPERAND_IMM32,
```

Comment:
Since allowing 32-bit immediates for RV32 is common and the MCInst will always store int<32>, should this be `OPERAND_SIMM32`?